src/java.base/share/classes/java/io/Reader.java

```diff
@@ -140,6 +140,208 @@ public void close() {
         };
     }
 
+    /**
+     * Returns a new {@code Reader} whose source is a {@link CharSequence}.
+     *
+     * <p> The returned reader's class provides an API compatible with
+     * {@link StringReader}, but with no guarantee of synchronization, and
+     * without limitation to {@code String} sources. This class is designed for
+     * use as a drop-in replacement for {@code StringReader} in places where the
+     * reader was being used by a single thread (as is generally the case).
+     * Where possible, it is recommended that this class be used in preference
+     * to {@code StringReader} as it will be faster under most implementations.
+     *
+     * <p> The returned stream is initially open. The stream is closed by
+     * calling the {@code close()} method. Subsequent calls to {@code close()}
+     * have no effect.
+     *
+     * <p> After the stream has been closed, the {@code read()},
+     * {@code read(char[])}, {@code read(char[], int, int)},
+     * {@code read(CharBuffer)}, {@code ready()}, {@code skip(long)}, and
+     * {@code transferTo()} methods all throw {@code IOException}.
```

Comment:
I believe these 2 paragraphs are all redundant besides the initial `The returned stream is initially open.` sentence: those are duplicating information already available from `Reader::close` specification.

Closing `StringReader` is optional but its specification doesn't mention this. I think we don't need to mention if closing is optional, so that one sentence is sufficient.

Reply:
This text already exists in this very file, it is a 1:1 copy from `nullReader()`. It cannot be wrong for `of()` if it was correct for `nullReader()`. To stay consistent with existing JavaDocs in this very same file, we should not have different texts for the same cases.

Reply:
`nullReader` needs this because it specifies the open behavior of these methods. This API does not have to; we can check out how others think in the CSR review.

Reply:
I have forwarded the current proposal as-is to the CSR.